sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonsView.java

```diff
@@ -170,6 +187,11 @@ public void updateFilterComponents() {
 		// TODO replace with Vaadin 8 databinding
 		applyingCriteria = true;
 
+		if (criteria.getPersonAssociation() != null && !isPersonAssociationAllowed(criteria.getPersonAssociation())) {
+			PersonAssociation firstAllowedAssociation = getFirstAllowedPersonAssociation();
+			criteria.setPersonAssociation(firstAllowedAssociation != null ? firstAllowedAssociation : PersonCriteria.DEFAULT_ASSOCIATION);
```

Comment:
`PersonCriteria.DEFAULT_ASSOCIATION` gives access to `ALL` data, but should be blocked completely.

Reply:
Necessary because PersonCriteria validates that personAssociation is not null; add a comment why this is not a problem.